pipelines/Java/amazoncorretto-config.json

```diff
@@ -6,7 +6,7 @@
     "builders": [
         {
             "type": "docker",
-            "image": "maven:3.9.8-amazoncorretto-11-al2023",
+            "image": "debian:bookworm-20240701-slim",
```

Comment:
Since the file is now `pkr.hcl` you can delete this file

Reply:
yup, will do small fixup PR